zebedee-cms/src/main/java/com/github/onsdigital/zebedee/json/CollectionDescription.java

```diff
@@ -162,4 +166,36 @@ public ApprovalStatus getApprovalStatus() {
     public void setApprovalStatus(ApprovalStatus approvalStatus) {
         this.approvalStatus = approvalStatus;
     }
+
+    public Set<CollectionInstance> getInstances() {
+
+        if (this.instances == null) {
+            this.instances = new HashSet<>();
+        }
+
+        return instances;
+    }
+
+    public Optional<CollectionInstance> getInstance(String instanceID) {
+
+        if (this.instances == null)
+            return Optional.empty();
+
+        return this.instances.stream()
+                .filter(i -> i.getId().equals(instanceID)).findFirst();
+    }
+
+    public void addInstance(CollectionInstance instance) {
+        if (this.instances == null)
+            this.instances = new HashSet<>();
```

Comment:
add brackets around if the clause or move on to same line.

Reply:
done